src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3307,14 +3307,15 @@ static char* allocate_pages_individually(size_t bytes, char* addr, DWORD flags,
   return p_buf;
 }
 
-static size_t large_page_init_decide_size() {
+size_t os::win32::large_page_init_decide_size() {
   // print a warning if any large page related flag is specified on command line
   bool warn_on_failure = !FLAG_IS_DEFAULT(UseLargePages) ||
                          !FLAG_IS_DEFAULT(LargePageSizeInBytes);
 
 #define WARN(msg) if (warn_on_failure) { warning(msg); }
```

Comment:
make the macro to allow variable parameters
#define WARN(...) if (warn_on_failure) { warning(__VA_ARGS__); }

Reply:
i made this change in upstream PR https://github.com/openjdk/jdk/pull/20758
also validated the gtest